.cookiecutter/includes/tox/setenv

```diff
@@ -0,0 +1 @@
+SQLALCHEMY_SILENCE_UBER_WARNING=1
```

Comment:
This isn't doing anything right now, but I forgot it on another PR. We'll need it sooner or later.